client/main.lua

```diff
@@ -1,29 +1,13 @@
-local lockpickCallback = nil
+local function openLockpick(cb)
+    cb(lib.skillCheck(Config.Difficulty))
+end
 
-AddEventHandler('qb-lockpick:client:openLockpick', function(callback)
-    lockpickCallback = callback
-    openLockpick(true)
-end)
+-- Support for basic qb-lockpick
+AddEventHandler('qb-lockpick:client:openLockpick', openLockpick)
 
-RegisterNUICallback('callback', function(data, cb)
-    openLockpick(false)
-    if lockpickCallback then
-        lockpickCallback(data.success)
-    end
-    cb('ok')
-end)
+-- Support for qb-lock forks like NoPixel
+AddEventHandler('kwk-lockpick:client:openLockpick', openLockpick)
```

Comment:
idk if its a great idea to just throw in Event Handlers for random scripts, wtf is `kwk-lockpick` and why should we have that event handler